### ISSUE TITLE
change name to bdrc-test-FolderName-SUMMARY-date-time

### ISSUE THREAD

**Issue by TBRC-JimK** [buda-base/asset-manager#34]
**Is your feature request related to a problem? Please describe.**
Users can't easily locate the results for a certain test. The log files are created for each run, and our users aren't big find and grep fans.

**Describe the solution you'd like**
Have each folder tested generate a separate log file, identifiable by the work. 
Have the file contain summary and detailed texts

[https://www.codota.com/code/java/methods/org.apache.log4j.FileAppender/setName](https://www.codota.com/code/java/methods/org.apache.log4j.FileAppender/setName)

[https://stackoverflow.com/questions/15441477/how-to-add-log4j2-appenders-at-runtime-programmatically](https://stackoverflow.com/questions/15441477/how-to-add-log4j2-appenders-at-runtime-programmatically) . This one has all the namespaces spelled out.

And also
https://www.programcreek.com/java-api-examples/?class=org.apache.logging.log4j.core.Logger&method=addAppender

**Describe alternatives you've considered**
Cloud alerts, database status logging

**Additional context**

**Comment by TBRC-JimK** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [clean up output](https://trello.com/c/eZa83mIQ/64-clean-up-output)

**Comment by TBRC-JimK** [comment]
Almost there. Don't forget to change 
```
jimk@Tseng:Logs$ cat W1FPL1554 
id,test_name,outcome,detail_path,error_number,error_test
/Users/jimk/dev/tmp/pub/00/W1FPL1554,Passed,No Files in Root Folder
/Users/jimk/dev/tmp/pub/00/W1FPL1554,Passed,Web Image Attributes
/Users/jimk/dev/tmp/pub/00/W1FPL1554,Passed,No folders allowed in Image Group folders
/Users/jimk/dev/tmp/pub/00/W1FPL1554,Passed,File Sequence Test
```
to 
- headers: id,test_name,outcome,error_number,error_test,detail_path
- Test_name first, outcome,  right pad long test name, then have empty fields with detail_path
-  For error details, leave id,test_name,outcome, long test name  blank, just have error_number, error_test, detail path

**Comment by TBRC-JimK** [comment]
Fixed in #44
